INSTALLATION.md

```diff
@@ -0,0 +1,48 @@
+## Development ##
+There are several methods for building and setting up a development environment.
+  
+### Manual Method ###
+1. Create a new virtualenv with Python 3.6 (`virtualenv --python=<path-to-py3> venv`) and source it
+2. Install requirements via pip. For the moment there are two options:
+   * Install with plain pip using the `requirements`
+   * Install using Pipenv (which reads from the Pipfile)
+3. Build nativepython libraries using `python setup.py build`
+4. Append the root of this repository to your `PYTHONPATH`
+  
+### Pipenv Method ###
+1. (Optional) Create a new virtualenv with Python 3.6 (`virtualenv --python=<path-to-py3> venv`) and source it. If you choose to use Pipenv alone, it will create an appropriate virtualenv for you.
+2. Run `pipenv install`
+3. Append the root directory of this repository to your `PYTHONPATH`
```

Comment:
I think that this 3rd step is not necessary because `nativepython` is installed in development mode (see the line in `Pipfile` that reads `nativepython = {editable = true,path = "."}`.

Reply:
Yes, true, that makes sense. Will change.